CRM/Uimods/Tools/BankAccount.php

```diff
@@ -46,6 +51,44 @@ public static function renderForm($formName, &$form) {
         $form->assign('viewCustomData', $viewCustomData);
       }
     }
+
+    if ($formName == 'CRM_Custom_Form_CustomDataByType') {
+      if ($form->getVar('_type') == 'Contribution' && $form->getAction() == CRM_Core_Action::UPDATE) {
```

Comment:
Let's also do it for `CRM_Core_Action::ADD`. Probably requires some changes in the following code.